src/compiler-rt

```diff
@@ -1 +1 @@
-Subproject commit 57315f7e07d09b6f0341ebbcd50dded6c20d782f
+Subproject commit e3b7e473a467c3b8c54c413526818c87d38aebc2
```

Comment:
This is fine by me, but as a heads up it's probably gonna make this PR pretty rough to land :(